test/test_grammars.rb

```diff
@@ -11,29 +11,27 @@ class TestGrammars < Minitest::Test
   ].freeze
 
   HASH_WHITELIST = [
-    "2edac46b0a63309c96442d2826321a442217472f", # Agda.tmbundle
-    "4da01d631a29c76456fd0bd16749c71e8d5f6dbf", # ant.tmbundle
-    "79e72fd673dcebadd8fbace8d43db3da96d2c09f", # bro-sublime
+    "4b4c1b4aff11b99e53091f95b24093655c0d9ed2", # Agda.tmbundle
+    "a92c0e4e644206078a158d663b9f033d0d0aefbf", # ant.tmbundle
+    "98eac91395c97ae090113e1b4527cae3228d563a", # bro-sublime
```

Comment:
These changes don't look related to your pull request. I guess you messed something up while updating your branch. Please remember to check what you are comtting before committing.

Reply:
You told me to update my branch. @pchaigno

Reply:
Yes. It looks like it wasn't properly updated though, or changes to `test/test_grammars.rb` wouldn't appear. How did you update it?

Reply:
@pchaigno by hand

Reply:
I mean what commands?

Reply:
Oh, I see. That's not how you update a branch :-)
Please see [this SO answer](https://stackoverflow.com/a/3877000/6884590) for two methods to update a branch (merge or rebase). I'd recommend you revert the changes you made and apply one of these methods.

Reply:
@pchaigno I see, that repo was a little bit messy.
I've cloned fresh one and made only mine changes.